### PR TITLE
fix(#1237): replicate native dropdown keyboard behaviour in non-nativ…

### PR DESCRIPTION
**Pull request by chrisolsen**
…e component

### CODE REVIEW

**Comment by Radha-Alberta** [comment]
- [ ] without opening dropdown, arrow keys should change the dropdown values.
- [ ] page scrolls while using arrow keys.
- [ ] accessibility: while selecting dropdown values, it doesn't speak the value selected.

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 1]
```diff
@@ -203,73 +199,182 @@
...
+      _el.removeEventListener("keydown", onInputKeyDown);
+    }
   }
 
   function onHighlight(e: Event) {
     _highlightedIndex = Number((e.target as HTMLElement).dataset.index);
   }
 
   function onNativeSelect(e: Event) {
+    console.log('in the on native select')
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:377]: Remove this

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 2]
```diff
@@ -447,12 +557,13 @@
...
     border: none;
+    font: var(--goa-font-family-sans);
     background-color: transparent;
     color: var(--goa-color-text-default);
     font-size: var(--goa-font-size-4);
     appearance: none;
-    padding: calc(var(--goa-space-xs) + 2px);
-    padding-left: 0.5rem;
+    padding: calc(var(--goa-space-xs) + 1px);
+    padding-left: 0.75rem;
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:566]: Make sense to change this to `goa-space-s`?

**Review by ArakTaiRoth — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**